src/system.js

```diff
@@ -141,22 +153,62 @@ module.exports = AFRAME.registerSystem('physics', {
   tick: function (t, dt) {
     if (!this.initialized || !dt) return;
 
+    const wrapperStartTime = Date.now();
+
     var i;
     var callbacks = this.callbacks;
 
     for (i = 0; i < this.callbacks.beforeStep.length; i++) {
       this.callbacks.beforeStep[i].beforeStep(t, dt);
     }
 
+    const engineStartTime = Date.now();
+
     this.driver.step(Math.min(dt / 1000, this.data.maxInterval));
     
+    const engineEndTime = Date.now();
```

Comment:
It there is reason you didn't use `performance.now()`?
From the documentation
https://developer.mozilla.org/en-US/docs/Web/API/Performance/now
it would be more precise apparently.

Reply:
ignorance :-)  Happy to update to get more precision